### DISCUSSION TITLE
Help with working of dropdown

### DISCUSSION THREAD

**Discussion by NirnayK** [shadcn-ui/ui#2778] [Q&A]
The problem i am facing is lets say i have 2 dropdown menu. If i click on the first one and if i want switch over to the second one i need to make 2 clicks instead of one since with the first click it comes out of the dropdown 1 and then on second click it activates drop down 2.

**Comment by pablotanner** [comment]
I think setting modal=false in the Dropdown.Root component (you need to define it yourself, but you can just look at how they did it for Dropdown.Content) might do the trick
